src/components/NavBar/NavBar.tsx

```diff
@@ -1,13 +1,17 @@
 import HomeIcon from "@mui/icons-material/Home";
 import MailIcon from "@mui/icons-material/Mail";
 import AccountCircleIcon from "@mui/icons-material/AccountCircle";
+import HomeOutlinedIcon from "@mui/icons-material/HomeOutlined";
+import MailOutlinedIcon from "@mui/icons-material/MailOutlined";
+import AccountCircleOutlinedIcon from "@mui/icons-material/AccountCircleOutlined";
 import { Avatar, Box, Button, List, Toolbar } from "@mui/material";
 import AccountMenu from "./AccountMenu";
 import NavItem from "./NavItem";
 import PostButtonModal from "./PostButtonModal";
 import { useState } from "react";
 import ComposePost from "../Posts/ComposePost";
 import { useAppSelector } from "../../state/hooks";
+import { useLocation } from "react-router-dom";
 
 const styles = {
   icon: {
```

Comment:
The font size of the icons seems a little too big to me, do you think adding `fontSize: "30px"` here would make it look better?

Reply:
Done